orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/config/LambdaConfigurationProperties.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 JPMorgan Chase & Co.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@ConfigurationProperties(prefix = "lambda")
```

Comment:
FWIW [the plugin](https://github.com/spinnaker-plugins/aws-lambda-deployment-plugin-spinnaker/blob/ae9aca40cfce86d43b0d372939ae80e1a45a82ba/lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/Config.java#L8) uses `lambdaPluginConfig`.  I think this change is totally fine.